layer/tests/CMakeLists.txt

```diff
@@ -49,7 +49,13 @@ function(LayerTest NAME)
                    layer_tests_main.cpp
                    vktestframework.cpp)
     add_dependencies(${TEST_NAME} ProfilesLayer ${TEST_JSON_FILES})
-    target_link_libraries(${TEST_NAME} Vulkan::Headers Vulkan::Vulkan GTest::gtest GTest::gtest_main Vulkan::LayerSettings)
+    target_link_libraries(${TEST_NAME}
+        Vulkan::Headers
+        Vulkan::LayerSettings
+        GTest::gtest
+        GTest::gtest_main
+        $<TARGET_NAME_IF_EXISTS:Vulkan::Loader>
```

Comment:
Why do we need this? The logic is different from before the PR

Reply:
Didn't realize this function wasn't used on Android. I'll remove the `TARGET_NAME_IF_EXISTS`

Reply:
fixed